### PR TITLE
Unarmed changes

### PR DESCRIPTION
**Pull request by ghost**
:cl: Boltersam
tweak: Brutes, Spartans, Orions and Elites now do higher unarmed damage, listed in descending order. Accompanied by specific attack text.
tweak: Grunts and both kig'yar species have unique attack text. Kig'yar species now have sharp and edged unarmed attacks.
tweak: critical unarmed attacks no longer have their chance tied to attack damage. They are now fixed at 5% chance to occur.
/:cl:

### CODE REVIEW

**Comment by BDpuffy420** [comment]
Have you even tested this?

**Comment by ghost** [comment]
I tested it on a private server, all the new attack texts work as intended, and all of them do the right amount of damage.

File: code/modules/halo/species/spartan.dm [hunk 1]
```diff
@@ -83,3 +84,10 @@
 
 /datum/species/spartan/sanitize_name(name)
 	return sanitizeName(name, allow_numbers=TRUE)
+
+/datum/unarmed_attack/spartan_punch
+    attack_verb = list("jabs", "slams", "punches", "knees", "kicks", "strikes")
+    attack_noun = list("fist")
```
Comment (X0-11) [inline code/modules/halo/species/spartan.dm:90]: verb/noun inconsist. "kicks" with their "fist" ?

**Review by X0-11 — DISMISSED** [review]

File: code/modules/halo/species/spartan.dm (hunk 1 above)
Comment (ghost) [reply code/modules/halo/species/spartan.dm:90]: in practice doesn't actually cause text "kicks with their fist", 
It uses "X [attack_verb] Y in their [target_limb]"
so e.g. "X kicks Y in the upper body!" "X knees Y in the head!"

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**